test/controllers/organizations/staff/fosterers_controller_test.rb

```diff
@@ -39,4 +40,49 @@ class Organizations::Staff::FosterersControllerTest < ActionDispatch::Integratio
       end
     end
   end
+
+  context "#edit" do
```

Comment:
I think we can remove these tests testing authorization now you have added the policy tests?

Reply:
oh, okay, I wanted to recreate what was written for the index action in this tests.